linode/manager
# refactor: [M3-7893] – Post-GA VPC cleanup

## Description 📝
Since VPC entered GA several weeks back, we are now able to circle back to complete some code cleanup. This ticket is primarily for `@TODO VPC` items related to the VPC feature flag and related conditionals.

## Changes  🔄
- Clean up `flags.vpc`, `showVPCs`, and related logic throughout many files
- Remove `manuallyUpdateConfigs` prop from `PowerActionsDialogOrDrawer.tsx`
- Use `import type { ...}` convention in a few places

## Target release date 🗓️
04/01/2024

## How to test 🧪
### Verification steps
Ensure there have been no regressions compared to prod, specifically:

- VPC appears in Create button dropdown and in PrimaryNav
- VPC landing and detail pages load properly
- VPC appears in list when you hit `Ctrl` + `Shift` + `K`
- For Linodes assigned to a VPC...
  - "VPC" section is present in the header panel on the Linode Details page
- For Linode Create flow...
  - VLAN accordion is present
  - `"Use Private IP for a backend node to a NodeBalancer. Use VPC instead for private communication between your Linodes."` copy is present for Private IP add-on
- Powering Linodes on and off
- `InterfaceSelect` works as expected in Linode Config dialog and VLAN accordion
- "VPC" appears in "Add Personal Access Token" drawer & "View Scopes" drawer for PATs
- No adverse impact to VPC Subnets table

## As an Author I have considered 🤔
